.babelrc

```diff
@@ -0,0 +1,19 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "current"
+      },
+      "loose": true,
+      "useBuiltIns": true
+    }]
+  ],
+  "plugins": [
+    "transform-async-generator-functions",
+    "transform-flow-strip-types",
+    ["transform-object-rest-spread", { "useBuiltIns": true }],
+    "transform-class-properties",
+    "add-module-exports"
+  ]
+}
```

Comment:
didn't like my scaffolding ? :P

Reply:
(but seriously can we use @4c/babel-preset-4catalyzer ?)

Reply:
wasn't ready yet :P

Reply:
my text got swallowed. no scaffolding is fine , but can we use the 4c babel-preset here